ansible/www-standalone/tools/promote/promote_release.sh

```diff
@@ -37,6 +37,8 @@ while true; do
   if [ "X${yorn}" == "Xy" ]; then
       . ${__dirname}/_promote.sh $site $2
       nodejs-latest-linker /home/dist/${site}/release/ /home/dist/${site}/docs/
+      dstdir=/home/dist/${site}/docs
+      . ${__dirname}/upload_to_cloudflare.sh $site $2
```

Comment:
Why is this needed for releases but not the non-release (e.g. nightlies, v8-canary) builds?

Reply:
this line will actually become redundant once https://github.com/nodejs/release-cloudflare-worker/pull/39 is merged